src/element/QueryTable/model/QueryTable.ts

```diff
@@ -333,18 +334,21 @@ export abstract class QueryTable<
 		// fetch query builder
 		const k_query = await this.fetchQueryBuilder();
 
-		// execute query and download up to 300 rows
-		const a_rows = await k_connection.execute(k_query.paginate(300));
+		// execute query and download up to 160 rows
+		const a_rows = await k_connection.execute(k_query.paginate(160));
 
 		// build XHTML table
 		const a_fields = this.queryType.fields;
 		const f_builder = k_contents.builder();
-		const yn_table = f_builder('table', {}, [
+		const yn_table = f_builder('div', {"class": "table-wrap"}, [
+		    f_builder('table', {"class": "wrapped confluenceTable tablesorter tablesorter-default stickyTableHeaders"}, [
 			f_builder('colgroup', {}, a_fields.map(() => f_builder('col'))),
-			f_builder('tbody', {}, [
-				f_builder('tr', {}, a_fields.map(g_field => f_builder('th', {}, [
+			f_builder('thead', {"class": "tableFloatingHeaderOriginal"}, [
+				f_builder('tr', {}, a_fields.map(g_field => f_builder('th', {"class": "confluenceTh tablesorter-header sortableHeader tablesorter-headerUnSorted dateFormat-ddmmyyyy"}, [
```

Comment:
do we need dateFormat-ddmmyyyy? I was looking at the output of a vanilla confluence table (without the pagination plugin) and it didn't have that class, maybe that's something that's configured in the plugin and added afterwards?

Reply:
I checked and dateFormat-ddmmyyyy does look to come from the table/filter plugin, removed it from the above